### DISCUSSION TITLE
Tutorial 1: Basic System Administration: Example Output for `journalctl` and `systemdctl` Command

### DISCUSSION THREAD

**Discussion by nyameko** [chpc-tech-eval/scc#56] [Show and tell]
You can verify the dns servers configured for your system using `cat /etc/resolv.conf`

You can verify that you `NetworkManager` service has been correctly initialized and is running using `sudo systemctl status NetworkManager`

![linux_basic_networking_checking_resolvconf_file](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/d8396a3a-6ce7-4e5b-b721-50919bc3b27c)

**Comment by nyameko** [comment]
Make sure that you execute you commands **ON THE CORRECT TERMINAL**!

Do not confuse you local Windows PowerShell terminal on your laptops, with your head node's terminal shell prompt:

![headnode_systemd-networkd](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/61a9630a-df22-4963-abc2-c8b7f135ad5e)

**Reply by n-zwane** [reply]
Team Technocrats SCC 2025
<img width="944" height="1072" alt="admin" src="https://github.com/user-attachments/assets/e07a16df-1ba6-4f55-9ec4-c2c72833928f" />

**Comment by nyameko** [comment]
You can check the status of DNS services using the `systemd` service or `resolvectl`:


![headnode_systemd_resolved](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/f0d1c676-c874-4118-a378-a2780bdb1215)

![headnode_resolv_status](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/03157c5e-73a3-4f3a-ba69-ef51ab791e73)

**Comment by nyameko** [comment]
You can query your Linux distribution, kernel and system details using `uname -a`

![headnode_hostname_status](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/69587dcf-020f-423b-b30f-32e9afee4e45)

**Comment by prince-taker** [comment]
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/69721467/9f8bf6c9-97f6-4b83-8e72-d3d43fc4293f)

Ctrl Alt Elite

**Comment by Dc-Spears** [comment]
CPUT (DC Spears)
![cput systemctl](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175037009/90c70f75-b63e-4c85-aa3e-79bf6d146fcc)

**Comment by Cyphers-UFH** [comment]
UFH (Cyphers)

![Screenshot 2024-07-09 060521](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175048513/1f34dd28-ae0b-4824-919f-62dafcc11990)

**Comment by nexusWorkers** [comment]
!
nexus networkers
![Screenshot 2024-07-09 084621](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175092203/8240423d-79ff-4674-af8a-55fbb1af4749)

**Comment by MishkaSingh1** [comment]
**EEElite(UCT)**
![tut1](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175049735/b693d681-32b0-4f0b-8e72-ae2bba88106c)

**Comment by Talhahpatelia** [comment]
## Wits-A
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/29210701/4d16d582-3c3c-4f1c-a24a-470ec50778f3)

**Comment by CraftierLamb599** [comment]
## Wits-1
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/137203651/db0e5263-1791-4003-b4c3-dafd00e2fbe9)

**Comment by SfisoMncube** [comment]
**Khulisa IP-A**

![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/151764841/1196858e-446a-4a9b-ad6f-42086fe1e384)

**Comment by ADELAIDEMOGALE** [comment]
### **Khulisa Data**
![Screenshot (687) 1](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/166052302/4db10d64-ae6c-4569-84c1-71002e78805b)

**Comment by Rosemary2301** [comment]
Algorithm Avengers
![network](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/143855810/971a4e5d-4e33-4f05-aeaf-3ad079a1e149)

**Comment by lisabeeaan** [comment]
**HPC Wizards**
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/136716760/02937b96-d233-4bf8-a861-b74602c5206c)

**Comment by SizweMarole-21389** [comment]
Cybershield

![systemconf](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175023186/8fd7cbc6-e526-48a4-aa3e-6986647c87c1)

**Comment by hoperiley0806** [comment]
**Parallel Pioneers**

![WhatsApp Image 2024-07-12 at 02 51 11_9654fa08](https://github.com/user-attachments/assets/1098d780-b1dc-4ae7-b1ea-8fb32bf24127)

**Comment by 15kay** [comment]
hypernet

![image](https://github.com/user-attachments/assets/b409ce4f-f9b7-421d-a8c2-5353415ee05c)

**Comment by Kohzmik** [comment]
CUT_Team
![image](https://github.com/user-attachments/assets/38da443b-45dc-4587-afad-24ff12aac0ed)

**Comment by JedK721** [comment]
**Wits-i**

![Screenshot 2024-07-13 161719](https://github.com/user-attachments/assets/7bbc1a54-cbbf-4cb5-9971-29167b74e851)

**Comment by 15kay** [comment]
[image: image.png]
HYPERNET

**Comment by Iloke-Alusala** [comment]
0V1RCL0CK1D

![Image 2025-07-07 at 16 49](https://github.com/user-attachments/assets/c96be780-28d9-41e7-ba89-7fa8c2ab1703)

**Comment by MphephuSamuel** [comment]
computer champions

![Screenshot (982)](https://github.com/user-attachments/assets/ae6f6d4f-d921-44d3-88fd-164d7b8a7d4f)

**Comment by DCspearsv2** [comment]
DC Spears
![tut1 4](https://github.com/user-attachments/assets/bdea8c85-0618-4bed-81b7-b145d674c627)

**Comment by KamvaMj** [comment]
DC Daggers

![Screenshot 2025-07-07 192818](https://github.com/user-attachments/assets/23ee1ae8-6b0c-4f81-a2e2-5b243ce5762a)

**Comment by UNISA-SCT25** [comment]
![Tut1_sudo](https://github.com/user-attachments/assets/931b9e27-fc76-4d79-8cbc-c6487952b6a1)

Systemdctl

**Comment by UNISA-SCT25** [comment]
![Tut1_journal](https://github.com/user-attachments/assets/5b6b6bff-f70a-45ce-8759-83a6f89affb4)

journalctl

**Comment by Moagi99** [comment]
TUT_X

![Screenshot (4)](https://github.com/user-attachments/assets/4a4aa42e-6ca2-48ec-bde1-ce76b3d19db6)
![Screenshot (5)](https://github.com/user-attachments/assets/7340dfc0-b445-4a47-a647-125d1f76391e)

**Comment by OddBitLoaders1** [comment]
Oddbitloaders       
![Screenshot 2025-07-08 182408](https://github.com/user-attachments/assets/31a469ff-67c4-4240-900a-b5d16beed171)

**Comment by Swissisking21** [comment]
Techno geeks
![Screenshot (280)](https://github.com/user-attachments/assets/d16be230-3d54-498a-bfde-92e66e8dd917)
![Screenshot (279)](https://github.com/user-attachments/assets/f50cf5d2-9f5e-4100-8096-3636652d29f2)

**Comment by Swissisking21** [comment]
Techno Geeks 
![Screenshot (282)](https://github.com/user-attachments/assets/0844ac6c-e206-4d9d-be63-0995c756d5fb)

**Comment by MubeenD1** [comment]
Wits-1

![image](https://github.com/user-attachments/assets/3432d909-34e3-42c8-987d-a0f29e6493b4)

**Comment by Kevin-Ebrahim** [comment]
![Output-for-journalctl-and-systemdctl-Command](https://github.com/user-attachments/assets/85153266-b5e0-434f-af81-7213b4d80c85)
Wits-A

**Comment by Cloud-Strive** [comment]
runtime blinders

sudo journalctl

![Screenshot (12)](https://github.com/user-attachments/assets/50e919d6-bc8c-42eb-aebc-3b4337884a06)

sudo systemctl
![Screenshot (11)](https://github.com/user-attachments/assets/6384c91f-438c-4adf-9ef5-d407b58fa05e)

sudo systemctl status NetworkManager
![Screenshot (10)](https://github.com/user-attachments/assets/d432bd8e-5cdf-459f-901f-85b118a6452e)

Tutorial 1 HPL benchmark

![Screenshot (9)](https://github.com/user-attachments/assets/35b13db2-0a90-4eec-9380-ac143ef4c609)

**Comment by ShavirV** [comment]
tuksbyters
<img width="914" height="857" alt="image" src="https://github.com/user-attachments/assets/c698bb77-9164-44b4-beeb-7414a9fe9402" />

<img width="910" height="816" alt="image" src="https://github.com/user-attachments/assets/4b32111e-1883-4d4b-ae62-b72c344f5d4d" />

<img width="919" height="542" alt="image" src="https://github.com/user-attachments/assets/f7361595-cc5c-49f6-8dac-bc655686ace1" />

**Comment by CadmusKei** [comment]
Team Jargon (UWC) | SCC 2026 | Basic System Administration commands (Systemctl and NetworkManager)
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/378ccb83-7d3a-473d-b880-1e6b078bdd27" />

**Comment by ThapeloVundla** [comment]
<img width="1860" height="999" alt="Screenshot from 2026-07-06 15-09-38" src="https://github.com/user-attachments/assets/a2a5b65f-4a88-462b-a2bd-be7a37eed0f8" />

DC Spears

**Comment by saxs-14** [comment]
3ncryp+3d (UMP) sudo systemctl status systemd-networkd
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/4a9efdb2-7e7b-443b-bb2f-3e2b23c16b24" />

**Comment by RoarkeBailey** [comment]
<img width="1856" height="509" alt="Screenshot from 2026-07-06 16-05-05" src="https://github.com/user-attachments/assets/4ed485c8-6685-4797-93eb-fd4136a5746d" />

G3MIKNIGHTS output of systemctl using NetworkManager for Rocky 9

**Comment by UNISA-QC-QUEENS** [comment]
UNISA_QC_QUEENS
<img width="1302" height="684" alt="Screenshot from 2026-07-06 16-43-12" src="https://github.com/user-attachments/assets/c3601445-03ee-402a-9a4c-6020e7f7fc52" />

**Comment by Slindokuhle19** [comment]
UnizuluComputeX
<img width="1489" height="595" alt="image" src="https://github.com/user-attachments/assets/e1d054ca-4c02-486b-b8f2-cf53d820a523" />

**Comment by Slindokuhle19** [comment]
UnizuluComputeX
<img width="1854" height="1048" alt="image" src="https://github.com/user-attachments/assets/d945e095-4d5e-4c53-b14a-968cef08f1cf" />

**Comment by bytedebuggers-dotcom** [comment]
**Basic system administration commands {systemctl}**
<img width="1302" height="408" alt="Screenshot From 2026-07-06 22-45-59" src="https://github.com/user-attachments/assets/64d4360a-680c-46ac-a541-b59adfa62faf" />

**Comment by mabotenick97-cloud** [comment]
CyberOps
<img width="1126" height="673" alt="Screenshot from 2026-07-07 12-45-45" src="https://github.com/user-attachments/assets/9bee38bc-a1a9-4915-8e5a-f4bad2044370" />
<img width="469" height="385" alt="Screenshot from 2026-07-07 12-46-09" src="https://github.com/user-attachments/assets/572a9c3f-863b-42c4-a9a1-524de7390ae1" />
<img width="883" height="896" alt="Screenshot from 2026-07-07 12-48-16" src="https://github.com/user-attachments/assets/0d50d8df-3e43-4f3e-a31e-f025350d369e" />
<img width="855" height="337" alt="Screenshot from 2026-07-07 12-48-43" src="https://github.com/user-attachments/assets/f628b665-396b-4dbf-832e-85683d69e66d" />
<img width="1850" height="779" alt="Screenshot from 2026-07-07 12-56-12" src="https://github.com/user-attachments/assets/01e39648-8219-4a97-b666-061ec2d1c7b9" />

problems we encounter " sudo systemctl status system-networkd" isnt for Rocky , therefore we used " sudo systemctl status NetworkManger"

**Comment by switchsquad** [comment]
<img width="1918" height="1001" alt="image" src="https://github.com/user-attachments/assets/4c411c7c-8c3b-49b9-949a-f4b708389f83" />


Switch Squad (TUT)

**Comment by JayJay-lab** [comment]
Sol_Plaatje_A SCC 2026     
<img width="1912" height="1082" alt="image" src="https://github.com/user-attachments/assets/524cdc4b-8db0-42f1-9f8f-8f4b1bad45a3" />

**Comment by mpho21925** [comment]
Ampere Alliance | SCC 2026
verifying some system's configuration settings

<img width="960" height="1020" alt="image" src="https://github.com/user-attachments/assets/88846c43-fb59-4384-a3f8-74cac005310e" />

**Comment by Leo-OB** [comment]
## DC Axes (CPUT) | SCC 2026
### Outputs of basic system administration commands `journalctl` and `sudo systemctl status NetworkManager`

<img width="1920" height="1080" alt="journalctl" src="https://github.com/user-attachments/assets/3da38be7-a16d-4f51-82dc-6b10686a12f6" />
<img width="1920" height="562" alt="T1_systemctl_status" src="https://github.com/user-attachments/assets/89d41ce9-f611-41fc-803a-00e39b9c7d5f" />

**Comment by majaha8** [comment]
Team Nebula
<img width="921" height="1004" alt="tutorial 1" src="https://github.com/user-attachments/assets/6bedaaee-52fb-4f36-9118-27177513f882" />

**Comment by Lihle0512** [comment]
ChaosCompilers

<img width="1857" height="609" alt="Screenshot from 2026-07-09 15-46-55" src="https://github.com/user-attachments/assets/323d5668-c65c-40d2-bfc4-d1ff61e3895f" />

**Comment by LufunoMiles** [comment]
Digital Dynamos  [ Basic System Administration: Example Output for `journalctl` and `systemdctl` Command
#56] <img width="1023" height="902" alt="basic admins" src="https://github.com/user-attachments/assets/6c1eefc7-5f86-44c3-bfe1-17e3a2d75e5e" />

**Comment by Leeu-Fusi** [comment]
Cluster Commanders | CUT

<img width="1195" height="607" alt="Screenshot from 2026-07-10 12-31-02" src="https://github.com/user-attachments/assets/7a9a47b7-c5fe-41ba-bced-2551067a0ce1" />

**Comment by MrLampies** [comment]
DC Spears journalctl systemdctl

<img width="1860" height="999" alt="Screenshot from 2026-07-06 15-09-38" src="https://github.com/user-attachments/assets/be14d571-f669-44c2-9dca-860aec471441" />

**Comment by ShavirV** [comment]
UPtime
<img width="943" height="945" alt="image" src="https://github.com/user-attachments/assets/4bec536a-52c6-4c50-915c-d7f197338aaf" />

**Comment by VerilM** [comment]
Wits-1:  Networking

<img width="1862" height="661" alt="image" src="https://github.com/user-attachments/assets/bad0e500-0b8e-4571-9fde-7880ef0a41f9" />
